src/app/tests/integration/common.cpp

```diff
@@ -55,12 +57,6 @@ void InitializeChip(void)
     }
 }
 
-void ShutdownChip(void)
```

Comment:
This is intentional, currently 1. we cannot shutdown the CHIP stack gracefully, and 2. there seems no requirement to make CHIP can be gracefully shutdown since we don't have state that need to be persisted in this way, and 3. the issue is caused by race during shutdown -- we just finished all the things we care so we try to shutdown the CHIP, but CHIP has a few more things to do (in fact, it is doing the routine of GC), this won't be an issue on real devices since we always expect the program will continue in that scenario.

Reply:
> there seems no requirement to make CHIP can be gracefully shutdown

The stack is supposed to be equally supportable on non-constrained devices that run multiple processes and DO start and stop programs.

Reply:
That's true, however, when taking look at the actual shutdown logic, it just do a few memory cleanup -- like releasing objects, which is unnecessary when we want to quit the program.

Reply:
Agree, we should make sure Shutdown works. It's a pretty expected command on controller side devices.